conf/claims.routes

```diff
@@ -12,9 +12,12 @@ GET         /bulk/enter-movement-reference-number                   uk.gov.hmrc.
 POST        /enter-movement-reference-number                        uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterMovementReferenceNumberController.enterMrnSubmit()
 
 GET         /change-movement-reference-number                       uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterMovementReferenceNumberController.changeMrn()
-GET         /bulk/echange-movement-reference-number                 uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterMovementReferenceNumberController.changeBulkMrn()
+GET         /bulk/change-movement-reference-number                 uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterMovementReferenceNumberController.changeBulkMrn()
```

Comment:
aren't these redundant now?

Reply:
oops. old commit

Reply:
These are not in the latest version of the code